src/mscorlib/src/System/Diagnostics/Stacktrace.cs

```diff
@@ -594,7 +603,9 @@ internal String ToString(TraceFormat traceFormat)
                         }
                     }
 
-                    if (sf.GetIsLastFrameFromForeignExceptionStackTrace())
+                    if (sf.GetIsLastFrameFromForeignExceptionStackTrace() &&
+                        declaringType?.Assembly != s_corelibAssembly && // Skip EDI boundary for mscorlib
```

Comment:
Why is this disabled for all of corelib?

Reply:
😞 

Will have to narrow further

Reply:
Perhaps attribute on method is way forward?

Reply:
attribute seems reasonable, and matches the pattern we used for StackTraceHidden handling

Reply:
Just switched it off for async; can address other areas if/when they arise